src/routes/embedded/auth/auth-more-providers/auth-more-providers.view.tsx

```diff
@@ -1,40 +1,93 @@
-import { DevFigmaScreen } from "~components/dev/figma-screen/figma-screen.component";
 import { useEmbedded } from "~utils/embedded/embedded.hooks";
 
-import screenSrc from "url:/assets-beta/figma-screens/auth.view.png";
+import {
+  AppleIcon,
+  Box,
+  Button,
+  Card,
+  FacebookIcon,
+  Row,
+  Text,
+  TwitterIcon,
+  WanderIcon
+} from "~components/embed";
+import { useCallback, useState } from "react";
+import type { AuthProviderType } from "embed-api";
 
 export function AuthMoreProvidersEmbeddedView() {
-  const { authenticate } = useEmbedded();
+  const { authenticate, authStatus } = useEmbedded();
+
+  const [selectedAuthProviderType, setSelectedAuthProviderType] =
+    useState<AuthProviderType | null>(null);
+
+  const areButtonsDisabled =
+    authStatus === "unknown" ||
+    authStatus === "loading" ||
+    authStatus === "authLoading" ||
+    !!selectedAuthProviderType;
 
   // TODO: Remember last selection and highlight that one / show it in the main screen (not in "More")
 
+  const handleAuthenticate = useCallback(
+    async (authProviderType: AuthProviderType) => {
+      setSelectedAuthProviderType(authProviderType);
+      await authenticate(authProviderType);
+      setSelectedAuthProviderType(null);
+    },
+    []
+  );
+
   return (
-    <DevFigmaScreen
-      title="More options"
-      src={screenSrc}
-      config={[
-        {
-          label: "Email & Password",
-          onClick: () => authenticate("EMAIL_N_PASSWORD")
-        },
-        {
-          label: "Facebook",
-          onClick: () => authenticate("FACEBOOK")
-        },
-        {
-          label: "Apple",
-          onClick: () => authenticate("APPLE")
-        },
-        {
-          label: "X",
-          onClick: () => authenticate("X")
-        },
-        {
-          label: "Back",
-          to: "/auth",
-          variant: "secondary"
-        }
-      ]}
-    />
+    <Card
+      headerText="Sign Up or Sign In"
+      subtitle="Select a method to authenticate"
+      footerElement={
+        <Row>
+          <Text variant={"bodyXs"} style={{ marginBottom: 0 }}>
+            {"Secured by"}
+          </Text>
+          <WanderIcon color="#838383" />
+        </Row>
+      }
+      hasBackButton={true}
+      onBackButtonClick={() => {
+        window.history.back();
+      }}
+      //   hasCloseButton={false}
+      size="auto"
+    >
+      <Box>
+        <Button
+          variant="outlined"
+          isFullWidth
+          icon={<FacebookIcon fontSize={24} />}
+          onClick={() => handleAuthenticate("FACEBOOK")}
+          isLoading={selectedAuthProviderType === "FACEBOOK"}
+          isDisabled={areButtonsDisabled}
```

Comment:
I think I'm introducing a but by having both `isLoading` and `isDisabled`. I'll check it out better tomorow.